core/src/OpenMPTarget/Kokkos_OpenMPTarget_Parallel.hpp

```diff
@@ -149,8 +155,9 @@ class OpenMPTargetExecTeamMember {
       }
 #pragma omp barrier
     }
-    return team_scratch[0];
+    value = team_scratch[0];
```

Comment:
Shouldnt this have given a warning at-least since it is now a `void` function.

Reply:
Discussed offline that an earlier change in this pull request missed writing back to `value` when removing the `return` statement.

Reply:
nvm wrong question